test/common.sh

```diff
@@ -51,3 +51,10 @@ function knative_setup() {
     wait_until_pods_running knative-eventing || return 1
   fi
 }
+
+# Create test resources and images
+function test_setup() {
+  echo ">> Uploading test images..."
+  ${REPO_ROOT_DIR}/test/upload-test-images.sh || return 1  
+}
+
```

Comment:
```suggestion
```

Reply:
@navidshaikh ok it take it out, previously the code formatter which was in place used to enforce and extra line :-)

Reply:
sorry there are 2 extra lines, removing one